tests/certification/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.19
 require (
 	github.com/dapr/components-contrib v1.9.0-rc.1
 	github.com/dapr/dapr v1.9.0-rc.3
-	github.com/dapr/go-sdk v1.5.1-0.20221004175845-b465b1fa0721
+	github.com/dapr/go-sdk v0.8.1-0.20200925214701-2d699b6c653f
```

Comment:
I'm just now noticing this, but why are we using such an old version of the Go SDK in this PR? I would have expected us to use the latest v1.5.x version.